tools/api-markdown-documenter/docs/lint-api-model.md

```diff
@@ -0,0 +1,32 @@
+`api-markdown-documenter lint-api-model`
```

Comment:
Generated by `oclif`

Reply:
Hmm. I like that in theory but seems odd to have a "lint" verb that _only_ works with a noun target. That said, if that's what we're doing elsewhere, I will always err on the side of consistency. I'll look into this.